minutes/minutes-2015-03-12.md

```diff
@@ -0,0 +1,368 @@
+
+Coordinating with Data Carpentry: Greg will talk to Tracy Teal in Cold Spring Harbor
+
+JW: Has posted a budget as a google doc, to avoid issues with excel etc.
+
+# ## Updates from Subcommittees 
+
+Fee subcomittee: Has had a meeting with Camille, working on readying a suggestion that the Steering committee can look at. This committee is likely to want to discuss this further with the Scientific Advisory Board to ensure that we get this right. We may also request more help from the community, there might be people out there who have more experience with working with minority and student groups that we could draw on. We are wondering how this works outside of US/UK.
+
+Finance: JW showed the spreadsheet showing projections for 2015, which includes projections for both income and expenses. Currently, it looks like we will make a small surplus at the end of the year, this is with currently estimating 8 fee-generating workshops per month. Greg noted that this might be a tad optimistic. We will have to work on generating more fee-paying workshops.
+
+Partner relations: JW told us that he as a comittee up and running. They are collecting ideas using trello.
+
+Assessment: JW told that they are approximately 10 people so far. Things are moving along nicely. 
+
+Communications: AH says she has a good group going, and they are ddealing with things via issues on github. 6-7 people so far. Outstanding issues are how to create an annual reports and how to manage our twitter account.
+
+# ## Corporate SWC Workshops
+
+GW: It is important that we deal with workshops for companies in a good way. The general consensus seems to be to charge companies more than our regular price, the big question here becomes how we differentiate entities.
+
+KL: Perhaps something to latch onto to distinguish between regular and more-than-regular price could be whether the company is open research/open code. Then, we give some benefit to the companies who will give it back to the community
+
+GW: What about academic groups who don’t share their material?
+
+AP: How do we know that these companies really publish everything as open source. 
+
+MD: Makes his living in a software company. Possibly do a pilot program, decide to do 5-10 corporate workshops and see how it goes. 
+
+GW: One way to justify higher price: The company buys effectively a couple of workshops, they get one, and the others would be put where we want them. This would effectively mean that a company donates workshops to others, and could also advertise with that fact.
+
+**Motion:** Katy/Matt: write up something for us to read about this next week, and this will be for next meeting. We will vote, send out on discuss list, and send out. 
+
+Motion passed.
+
+# ## Issues in the board repo
+
+These issues were voted on as described in the agenda above. 
+
+Regarding Robert´s rules: we still have to figure out what happens with votes cast, but not for or against a motion.
```

Comment:
I'm not sure I understand your winky face. 

The concern was how to count abstentions ( a real thing ), and as you point out, it is clear in Robert's Rules. We were in the middle of a meeting and didn't have it handy. We just made a note to look it up later. Since we were in the middle of the meeting, that didn't happen right that second. It has been cleared up. Please don't worry about this.